pkg/synopsysctl/cmd_create.go

```diff
@@ -190,13 +190,50 @@ var createAlertNativeCmd = &cobra.Command{
 	},
 }
 
+func checkPasswords(cmd *cobra.Command) {
```

Comment:
Move this function under the comment for `Create Black Duck Commands`

Reply:
done.